docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,6 +64,52 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
+## Email Sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the url.
+
+### Enable email sharing
+
+> **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
+
+Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+
+```
+[feature_toggles]
+publicDashboardsEmailSharing = true
+```
+
+### Inviting a viewer
+
+1. Click **Only specified people**.
+2. Enter the email you want to share the public dashboard with.
+3. Click **Invite**.
+4. They will receive an email with a one-time use link. This link must be used within **1 hour** or it will be considered expired. Once the link is used, it will give the viewer access to the public dashboard for **30 days**.
+
+### Viewers requesting access
+
+If a viewer without access tries to navigate to the public dashboard, they will be asked to request access by providing their email. They will receive an email with a new one-time use link if the email they provided has already been invited to view the public dashboard and has not been revoked.
+
+### Revoking access for a viewer
+
+1. Click **revoke** on the viewer you'd like to revoke access for.
+2. This will immediately prevent the user from accessing the public dashboard or using any existing one-time use links they may have.
```

Comment:
A few things here:

- Same thing as above, we want to add in the preliminary steps. 
- Is "revoke" lower case in the UI? 
- I've also made a small change to avoid repetition. 
- I've removed step 2 from the steps because it's not a step the user takes, but an outcome of a steps they've taken.
- 

```suggestion
1. Click **revoke** on the viewer you'd like to remove access for.

Immediately, the user no longer has access to the public dashboard, nor can they use any existing one-time use links they may have.
```

Reply:
Revoke and Resend are both uppercase
<img width="726" alt="image" src="https://user-images.githubusercontent.com/20256983/224311958-b8e8c77f-db90-4635-8da0-694102c7e270.png">